c2corg_ui/static/js/map/map.js

```diff
@@ -887,5 +889,16 @@ app.MapController.prototype.resetFeature = function() {
   this.scope_.$root.$emit('mapFeaturesChange', features, true /* isReset */);
 };
 
+/**
+ * @private
+ */
+app.MapController.prototype.updateMapResize_ = function() {
+  setTimeout(function() {
```

Comment:
Please rather use ``ngeo,Debounce`` as it's done at https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/map/map.js#L275-L278